src/v/cluster/data_policy_frontend.cc

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#include "cluster/data_policy_frontend.h"
+
+#include "cluster/cluster_utils.h"
+
+#include <optional>
+
+namespace cluster {
+
+data_policy_frontend::data_policy_frontend(
+  ss::sharded<controller_stm>& stm, ss::sharded<ss::abort_source>& as) noexcept
+  : _stm(stm)
+  , _as(as) {}
+
+ss::future<std::error_code> data_policy_frontend::set_data_policy(
```

Comment:
the names of these methods should match the names of commands, for consistency since they are 1-1 with the commands.